### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by xuchunyang**
- Add URL header
- Add Version header
- Replace cl with cl-lib, fix requirements
- Better default value of helm-wordnet-wordnet-location (for HomeBrew)
- Use helm macros to build source instead of build manually

### CODE REVIEW

**Comment by raghavgautam** [comment]
@xuchunyang Please rebase/merge.

**Comment by xuchunyang** [comment]
Done.

File: helm-wordnet.el [hunk 1]
```diff
@@ -72,7 +78,7 @@
 (defun helm-wordnet-wordnet-wordlist ()
   "Fetch WordNet suggestions and return them as a list."
   (let* ((all-indexes (directory-files helm-wordnet-wordnet-location t "index\\..*" ))
-	 (word-indexes (cl-remove-if (lambda (x) (string-match-p "index\\.sense$" x)) all-indexes)))
+         (word-indexes (cl-remove-if (lambda (x) (string-match-p "index\\.sense$" x)) all-indexes)))
```
Comment (raghavgautam) [inline helm-wordnet.el:81]: I am not sure if this change is required.

**Comment by raghavgautam** [comment]
Looks good overall. Merging.

File: helm-wordnet.el (hunk 1 above)
Comment (xuchunyang) [inline helm-wordnet.el:81]: It's because you are using Tab to indent and I changed it to Space.

**Pull request merged**